packages/web/src/common/store/pages/search-page/lineups/tracks/sagas.js

```diff
@@ -6,11 +6,12 @@ import {
 } from '@audius/common'
 import { select, all, call, getContext } from 'redux-saga/effects'
 
-import { LineupSagas, isMobile } from 'common/store/lineup/sagas'
+import { LineupSagas } from 'common/store/lineup/sagas'
 import {
   getSearchResults,
   getTagSearchResults
 } from 'common/store/pages/search-page/sagas'
+import { isMobileWeb } from 'common/utils/isMobile'
```

Comment:
i like the rename a lot. should we also change the isMobile file name?

Reply:
ooh yes!